cloud/amazon/ec2_vpc_igw.py

```diff
@@ -133,14 +131,14 @@ def main():
 
     if region:
```

Comment:
`region` should be added to the documentation too (it's required for modules that use the `ec2_argument_spec` rather than `aws_argument_spec`)